pyproject.toml

```diff
@@ -5,9 +5,8 @@ build-backend = "setuptools.build_meta"
 
 [tool.black]
 line-length = 88
-py36 = false  # Python 2 is still supported
+target-version = ['py36', 'py37', 'py38']
```

Comment:
target-version here includes py38, even though we don't support that in testing...

Reply:
This if for Black though:

```
  -t, --target-version [py27|py33|py34|py35|py36|py37|py38]
                                  Python versions that should be supported by
                                  Black's output. [default: per-file auto-
                                  detection]
```

so this isn't affecting the testing.